source/Pages/Notification/components/ImportToken/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation, initReactI18next } from 'react-i18next';
+import { PortRPC } from '@psychedelic/browser-rpc';
+import i18n from 'i18next';
+import { useDispatch } from 'react-redux';
+import {
+  Button,
+  Container,
+  IncomingAction,
+} from '@ui';
+import { Layout, TokenIcon } from '@components';
+import PropTypes from 'prop-types';
+import { CONNECTION_STATUS } from '@shared/constants/connectionStatus';
+import { setAccountInfo } from '@redux/wallet';
+import { HANDLER_TYPES, sendMessage } from '@background/Keyring';
+import initConfig from '../../../../locales';
+import useStyles from './styles';
+import { reviewPendingTransaction } from '@modules/storageManager';
+import DisplayBox from '../Sign/components/Details/components/DisplayBox';
+
+i18n.use(initReactI18next).init(initConfig);
+
+const portRPC = new PortRPC({
+  name: 'notification-port',
+  target: 'bg-script',
+  timeout: 20000,
+});
+
+portRPC.start();
+
+const ImportToken = ({ args, metadata, callId, portId, setOnTimeout, transactionId, }) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const [loading, setLoading] = useState('');
```

Comment:
why use a string here? 
shouldn't it be a bool?

Reply:
probably, just to difference if its loading the approve or decline and only show the loading there